melos.yaml

```diff
@@ -18,19 +18,19 @@ scripts:
       cp -n "\$MELOS_ROOT_PATH"/.circleci/dummy_amplifyconfiguration.dart lib/amplifyconfiguration.dart | true
 
   build:examples:ios: >
-    melos exec -c 1 --scope="*example*,sample_app" --fail-fast -- \
+    melos exec -c 1 --scope="*example*,sample_app" --ignore="*_android*" --fail-fast -- \
```

Comment:
*nit:* Once all the plugins are federated, the scope can be updated to include only ios sample apps, rather than having android excluded. Maybe worth a todo.

Something like - `melos exec -c 1 --scope="*ios_example*"`